androidScanner/README.md

```diff
@@ -1,8 +1,11 @@
-### Android scanner (Minimum Viable Product)
-Notable points and flaws that can be fixed in future:
-* Requires at least Android API 23
-* Doesn't use ubiMqtt (uses eclipse paho)
-* Handles all BLE signals the same way (doesn't differ beacons)
-* Doesn't work with regular bluetooth signals
-* Has a delay (1.5 - 2.5s) between scanning waves
-* Was tested only with one device (Galaxy S9) 
+### Beacon scanner for Android 6
+
+Installation for Android tablet
+
+* Download project
+* Change server adress, topic and abserverId if needed
```

Comment:
I think we should specify how the values are changed. Via application UI? Via command line arguments?

Reply:
More detailed guidance will be added later on when remote configuration from the terminal will be added.